rollup.config.mjs

```diff
@@ -1,8 +1,18 @@
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import commonjs from '@rollup/plugin-commonjs'
 import typescript from '@rollup/plugin-typescript'
+import { readFile } from 'fs/promises'
 
-const external = []
+const packageJSON = JSON.parse(
+	await readFile(
+		new URL('./package.json', import.meta.url)
+	)
+)
```

Comment:
You should be able to import the json straight away 
`import packageJSON from './package.json'`